roles/bindle-profiles/tasks/main.yml

```diff
@@ -16,26 +16,31 @@
   git: repo=https://github.com/CloudBindle/Bindle
        dest=/home/{{ user_name }}/architecture2/Bindle
        version={{ bindle_git_branch }}
+       force=no
```

Comment:
In my tests, this caused ansible to fail and stop when it tried to checkout the repository.